src/scripts.c

```diff
@@ -53,6 +53,15 @@ static void update_triggers(bool block)
 	char *block_flag = NULL;
 	__attribute__((unused)) int ret = 0;
 
+	/* Check that systemd is working (container case)
+	 * and return if it is not after printing an error */
+	ret = system("/usr/bin/systemctl &> /dev/null");
```

Comment:
The `&>` syntax is a bashism. If you need to redirect both stdout and stderr, you can do `> /dev/null 2>&1`.